features/lwipstack/lwipopts.h

```diff
@@ -159,6 +159,14 @@
 #define TCP_WND                     MBED_CONF_LWIP_TCP_WND
 #endif
 
+#ifdef MBED_CONF_LWIP_TCP_MAXRTX
+#define TCP_MAXRTX                  MBED_CONF_LWIP_TCP_MAXRTX
+#endif
+
+#ifdef MBED_CONF_TCP_SYNMAXRTX
```

Comment:
Isn't the _LWIP_ part missing in this define?

Reply:
Good catch...

Yes it is.